package.json

```diff
@@ -48,6 +53,16 @@
       "pre-commit": "lint-staged"
     }
   },
+  "mocha": {
+    "require": [
+      "reflect-metadata/Reflect",
+      "ignore-styles"
+    ],
+    "reporter": "spec",
+    "colors": true,
+    "watch-extensions": "js",
+    "timeout": 10000
+  },
```

Comment:
this configuration is kind of duplicated, I think we should stick with just one config (+ the launch.json) and find a way to attach FE tests to the yarn test command that gets called by the CI.